public/sass/custom/kth-klaro.scss

```diff
@@ -203,43 +123,76 @@
         }
       }
 
+      a {
+        color: #036eb8;
+        text-decoration: underline;
+        text-underline-offset: 0.25rem;
+      }
+
+      //Header section
       .cm-header {
         padding: 1.5rem;
 
-        h1 {
-          &.title {
-            font-size: 1.875rem;
-            font-weight: 400;
-            color: $black;
-            line-height: 1.3125;
-            margin-top: 0;
-            margin-bottom: 0.875rem;
+        .hide {
+          svg {
+            stroke: $black;
           }
         }
+
+        .title {
+          font-size: 1.875rem;
+          font-weight: 400;
+          color: $black;
+          line-height: 1.3125;
+          margin-top: 0;
+          margin-bottom: 0.875rem;
+        }
+
+        
       }
 
+      //Body section
       .cm-body {
         padding: 1.5rem;
 
-        ul {
-          &.cm-services {
-            li {
-              &.cm-service {
-                span {
-                  &.cm-list-title {
-                    color: $black;
-                    font-size: 1rem;
-                  }
-                  &.cm-required {
-                    color: $black;
-                    font-size: 1rem;
-                  }
+        .cm-services {
```

Comment:
I think we can remove this line since we don't have any css for cm-services and I don't think we need that high specificity